lib/create-environment-api.ts

```diff
@@ -159,6 +167,218 @@ export default function createEnvironmentApi(makeRequest: MakeRequest) {
       return wrapAsset(makeRequest, assetData)
     },
 
+    /**
+     *
+     * @description Get a BulkAction by ID.
+     *  See: https://www.contentful.com/developers/docs/references/content-management-api/#/reference/bulk-actions/bulk-action
+     * @param bulkActionId - ID of the BulkAction to fetch
+     * @returns - Promise with the BulkAction
+     *
+     * @example ```javascript
+     * const contentful = require('contentful-management')
+     *
+     * const client = contentful.createClient({
+     *   accessToken: '<content_management_api_key>'
+     * })
+     *
+     * client.getSpace('<space_id>')
+     * .then((space) => space.getEnvironment('<environment-id>'))
+     * .then((environment) => environment.getBulkAction('<bulk_action_id>'))
+     * .then((bulkAction) => console.log(bulkAction))
+     * ```
+     */
+    getBulkAction<T extends BulkActionPayload = any>(bulkActionId: string): Promise<BulkAction<T>> {
+      const raw = this.toPlainObject() // Environment object
```

Comment:
wonder if using `const raw: EnvironmentParams = this.toPlainObject()` would help

Reply:
(it does)